### PR TITLE
http: Fix issue #1693

### PR DESCRIPTION
**Pull request by joe-conigliaro**
This one fixes https://github.com/vlang/v/issues/1693

### CODE REVIEW

File: thirdparty/vschannel/vschannel.c [hunk 1]
```diff
@@ -201,6 +201,7 @@ INT request(INT iport, CHAR *host, CHAR *req, CHAR *out)
 		return resp_length;
 	}
 	
+	printf(out);
```
Comment (spytheman) [inline thirdparty/vschannel/vschannel.c]: Is this needed?

**Comment by Bowero** [comment]
I am also wondering what https://github.com/vlang/v/pull/1734/commits/c1694f6f25b22786e9040f1bf07da1b569e5ecd2 is for.

File: thirdparty/vschannel/vschannel.c (hunk 1 above)
Comment (joe-conigliaro) [reply thirdparty/vschannel/vschannel.c]: nah that was left from me testing haha

**Comment by joe-conigliaro** [comment]
@Bowero req_length wasn't obvious that it was required length, looked like request length, so i changed it to required.

File: thirdparty/vschannel/vschannel.c (hunk 1 above)
Comment (joe-conigliaro) [reply thirdparty/vschannel/vschannel.c]: removed thanks

**Comment by spytheman** [comment]
@Bowero I think they will get squashed during the merging into a single commit.

**Comment by medvednikov** [comment]
GitHub allows to do the squashing in a couple of clicks, that's what I always do.

**Pull request merged**